roundware/rw/models.py

```diff
@@ -390,39 +393,59 @@ def media_display(self):
 
     def audio_player(self):
         if self.mediatype == 'audio':
-            return """<div data-filename="%s" class="media-display audio-file"></div>""" % self.filename
+            #return """<div data-filename="%s" class="media-display audio-file"></div>""" % self.filename
```

Comment:
Remove all of these code removal comments. The git diff for this commit will show us what was removed if it needs to be replaced.